frontend/src/pages/agreements/list/AgreementTableRow.jsx

```diff
@@ -145,25 +157,27 @@ export const AgreementTableRow = ({ agreement }) => {
         return (
             <>
                 {(status === "Draft" || status === "In Review") && (
-                    <div className="display-flex flex-align-center">
+                    <div className="display-flex flex-align-center ">
```

Comment:
is the trailing space needed?

Reply:
nope, I was hoping `prettier would catch and fix this for me` 🤦🏿‍♂️